crates/eframe/src/web/events.rs

```diff
@@ -402,6 +402,22 @@ pub fn install_canvas_events(runner_container: &mut AppRunnerContainer) -> Resul
         &canvas,
         "wheel",
         |event: web_sys::WheelEvent, mut runner_lock| {
+            {
+                let unit = match event.delta_mode() {
+                    web_sys::WheelEvent::DOM_DELTA_PIXEL => egui::MouseWheelUnit::Point,
+                    web_sys::WheelEvent::DOM_DELTA_LINE => egui::MouseWheelUnit::Line,
+                    web_sys::WheelEvent::DOM_DELTA_PAGE => egui::MouseWheelUnit::Page,
+                    3u32..=u32::MAX => return,
+                };
+                let delta = egui::vec2(event.delta_x() as f32, event.delta_y() as f32);
+                let modifiers = runner_lock.input.raw.modifiers;
+
+                runner_lock.input.raw.events.push(egui::Event::MouseWheel {
+                    unit,
+                    delta,
+                    modifiers,
+                });
+            }
             let scroll_multiplier = match event.delta_mode() {
```

Comment:
you can exhaustively match on `unit` here instead now
```suggestion
            let scroll_multiplier = match unit {
```

Reply:
I tried to keep the raw event part self-contained to avoid introducing bugs, and to make it easy to remove or comment out if needed. But depending on the answer to the comment below about the target differences, it might make more sense to merge the two parts into one, indeed.